src/main/java/br/com/caelum/comidinhas/tipoCozinha/EditarTipoCozinhaValidator.java

```diff
@@ -0,0 +1,26 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.springframework.validation.*;
+
+class EditarTipoCozinhaValidator implements Validator {
+
+    TipoCozinhaRepository tipoCozinhaRepository;
```

Comment:
Aqui, o atributo poderia ser `private final`:

```suggestion
    private final TipoCozinhaRepository tipoCozinhaRepository;
```